### PR TITLE
Impute disclaimer

### PR DESCRIPTION
**Pull request by zrm22**
## Description

Adding in a Disclaimer about the imputation issues currently facing the PHG. 

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Adding in Imputation Disclaimer Note.
```

### CODE REVIEW

**Review by tcasstevens — APPROVED** [review]

File: docs/imputation.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Imputation
 
+## Imputation Disclaimer: Currently we are noticing issues related to the imputation process. We are noticing for some PHGs there are very low mapping rates(~20% WGS reads).  These low mapping rates are likely contributing to some severe errors during the path finding steps. We believe we have an idea of some initial fixes and are working to implement.  We will update the documentation as soon as we have a solution.  Users can still build a PHG without running into this issue. We apologize for any inconvenience this may cause.
```
Comment (lynnjo) [inline docs/imputation.md:3]: Some suggested changes:

I would remove the "We are noticing" in the second sentence - it is too repetitious.  Second sentence should start with "Some PHGs ..."

The sentence that begins with "We believe we have an idea of some initial fixes " seems too wordy and sounds too tentative.  I'd reword to something like "We have identified issues and are implementing initial fixes"

I think the second to last sentence ("Users can still build a PHG without running into this issue") might be clearer if you said  "This issue does not affect the building of the PHG, only imputation runs.  Existing PHG builds should be fine"

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # PHG version 2
 [![PHGv2 CI](https://github.com/maize-genetics/phg_v2/actions/workflows/phgv2_ci.yml/badge.svg)](https://github.com/maize-genetics/phg_v2/actions/workflows/phgv2_ci.yml) [![codecov](https://codecov.io/gh/maize-genetics/phg_v2/graph/badge.svg?token=4BVD2QXQ1A)](https://codecov.io/gh/maize-genetics/phg_v2) [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Imputation Disclaimer: Currently we are noticing issues related to the imputation process. We are noticing for some PHGs there are very low mapping rates(~20% WGS reads).  These low mapping rates are likely contributing to some severe errors during the path finding steps. We believe we have an idea of some initial fixes and are working to implement.  We will update the documentation as soon as we have a solution.  Users can still build a PHG without running into this issue. We apologize for any inconvenience this may cause.
+
```
Comment (lynnjo) [inline README.md:5]: see below

**Review by lynnjo — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (btmonier) [reply README.md:5]: Perhaps write this in a caution block and simplify:

```
> ![CAUTION]
> **Imputation disclaimer**: We are experiencing issues with imputation, where 
> some PHGs have low mapping rates (~20% with WGS reads), causing errors 
> in the path-finding steps. We are working on fixes and will update the 
> documentation once a solution is available. 
> 
> You can still build a PHG without encountering this issue. We apologize 
> for the inconvenience.
```

File: docs/imputation.md (hunk 1 above)
Comment (btmonier) [reply docs/imputation.md:3]: Similar to the prior statement but with mkdocs-flavored markdown:

```
!!! danger "Imputation disclaimer"
    We are experiencing issues with imputation, where some PHGs have low 
    mapping rates (~20% with WGS reads), causing errors in the path-finding steps. 
    We are working on fixes and will update the documentation once a solution 
    is available. 
     
    You can still build a PHG without encountering this issue. We apologize 
    for the inconvenience.
```

File: README.md (hunk 2 above)
Comment (bethecon) [reply README.md:5]: I would add to this: 

> ![CAUTION]
> **Imputation disclaimer**: We are experiencing issues with imputation, where 
> some PHGs have low mapping rates (~20% with WGS reads), causing errors 
> in the path-finding steps. We are working on fixes and will update the 
> documentation once a solution is available. 
> 
> You can still build a PHG without encountering this issue. We apologize 
> for the inconvenience. 
>
>PHGv2 is under active development and we expect to encounter bugs. 
>We aim to report bugs as soon as they are discovered.

Comment (zrm22) [reply README.md:5]: My only concern with a caution block vs what this is is that this is a h2 heading and is very large and up front.  I guess the information being at the top should be clear if people read... I will change it.

File: docs/imputation.md (hunk 1 above)
Comment (zrm22) [reply docs/imputation.md]: @btmonier Is it better to use mkdocs form or one that looks good on the ReadMe from the GitHub.  I updated it to match this form, but it looks bad from the GitHub page.  Any ideas?

Comment (btmonier) [reply docs/imputation.md]: @zrm22  - It's because mkdocs-material and GitHub markdown use different syntax on how to build admonition blocks. I use GitHub flavored markdown for the README since that is not rendered for the site.

**Comment by zrm22** [comment]
Ok Everything is rendering fine.  If @bethecon and @btmonier Can ok the revised message I will merge it in.

**Review by bethecon — APPROVED** [review]

**Review by btmonier — APPROVED** [review]
